actions/add-review-url/src/index.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import AddComment from './AddReviewUrl';
+import * as core from "@actions/core";
+
+async function run() {
+    const  addComment = new AddComment();

```

Comment:
Extra space between `const` and `addComment`. If the editor is not able to format this for you, try looking at tslint.

Reply:
Using tslint now. Removed the extra space.